lib/hca/enrollment_system.rb

```diff
@@ -85,7 +85,7 @@ def financial_flag?(veteran)
       veteran['understandsFinancialDisclosure'] || veteran['discloseFinancialInformation']
     end
 
-    def format_address(address)
+    def format_address(address, type_code = nil)
```

Comment:
Prefer keyword arguments to reduce ambiguity in method parameter orders, e.g.,

`def format_address(address, type_code: nil)`